docker-compose.yml

```diff
@@ -3,17 +3,25 @@ version: '3.8'
 services:
 
   localstack:
-    image: localstack/localstack:0.11.1
+    healthcheck:
+      test: "bash -c 'AWS_ACCESS_KEY_ID=fake AWS_SECRET_ACCESS_KEY=fake aws --endpoint-url=http://localhost:4566 s3 ls'"
```

Comment:
This healthcheck isn't right? You're querying s3 but notify doesn't use s3. Can this be changed to the health endpoint now?